README.md

```diff
@@ -1,887 +1,745 @@
-<!--

```

Comment:
it seems the entire file got changed...

Reply:
The reason is that I have also replaced CRLF with LF. I added a separated commit with only  CRLF to evidence changes